admin_manual/office/example-docker.rst

```diff
@@ -0,0 +1,118 @@
+
+    docker pull collabora/code
+    docker run -t -d -p 127.0.0.1:9980:9980 \
+        -e 'domain=cloud\\.example\\.com' \
+        --restart always \
+        --cap-add MKNOD \
+        collabora/code
+
+That will be enough. Once you have done that the server will listen on "localhost:9980". Now we just need to configure the locally installed Apache reverse proxy.
+
+2. Install the Apache reverse proxy
+***********************************
+
+On a recent Ubuntu or Debian this should be possible using:
+
+.. code-block:: bash
+
+    apt-get install apache2
+    a2enmod proxy proxy_wstunnel proxy_http ssl
+
+Afterward, configure one VirtualHost properly to proxy the traffic. For security reason we recommend to use a subdomain such as office.example.com instead of running on the same domain. An example config can be found below:
+
+.. code-block:: apache2
+
+    <VirtualHost *:443>
+        ServerName office.example.com:443
+
+        # SSL configuration, you may want to take the easy route instead and use Lets Encrypt!
+        SSLEngine on
+        SSLCertificateFile /path/to/signed_certificate
+        SSLCertificateChainFile /path/to/intermediate_certificate
+        SSLCertificateKeyFile /path/to/private/key
+        SSLProtocol             all -SSLv2 -SSLv3
+        SSLCipherSuite ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS
+        SSLHonorCipherOrder     on
+
+        # Encoded slashes need to be allowed
+        AllowEncodedSlashes NoDecode
+
+        # Container uses a unique non-signed certificate
+        SSLProxyEngine On
+        SSLProxyVerify None
+        SSLProxyCheckPeerCN Off
+        SSLProxyCheckPeerName Off
+
+        # keep the host
+        ProxyPreserveHost On
+
+        # static html, js, images, etc. served from loolwsd
+        # loleaflet is the client part of LibreOffice Online
+        ProxyPass           /loleaflet https://127.0.0.1:9980/loleaflet retry=0
```

Comment:
The migration docs mention that `loleaflet` has been renamed to `browser`, and `lool` to `cool`. So it seems like this example is outdated and should be migrated to the new names, no?